Allows building without Python installed, when tests are not enabled.

--- a/meson.build
+++ b/meson.build
@@ -59,11 +59,13 @@ endif
 
 bash = find_program('bash', required : false)
 
+if get_option('tests')
 if get_option('python') == ''
   python = find_program('python3')
 else
   python = find_program(get_option('python'))
 endif
+endif
 
 libcap_dep = dependency('libcap', required : true)
 
